interface/billing/new_payment.php

```diff
@@ -117,181 +117,165 @@
 
```

Comment:
Issues with this page are circled:
![image](https://user-images.githubusercontent.com/46497068/92337153-ad722400-f075-11ea-89c9-fe137ad42421.png)